### PR TITLE
Update Intent to Import

### PR DESCRIPTION
**Pull request by ipbyrne**
This PR is for updating the Intent to Import Credential required fields and examples.

### CODE REVIEW

**Comment by TallTed** [comment]
Please update the title of this PR to `Update Intent to Import`.
(Current last word is erroneously `Important`.)

File: docs/openapi/components/schemas/common/IntentToImport.yml [hunk 1]
```diff
@@ -27,15 +27,15 @@ properties:
     description: >-
       Organization that is declaring their intention for a product to be exported to a
       different location than where it was created.
-    type: object
+    $ref: ./Organization.yml
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/common/IntentToImport.yml:30]: I don't think we want to pull in the whole Organization object. We want to only include a subset of what we want to use. Example here: https://github.com/w3c-ccg/traceability-vocab/blob/intent-to-import/docs/openapi/components/schemas/credentials/MillTestReportCredential.yml#L99

File: docs/openapi/components/schemas/common/IntentToImport.yml [hunk 2]
```diff
@@ -27,15 +27,15 @@ properties:
...
     $linkedData:
       term: exporter
       '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#exporterParty
   importer:
     title: Importer
     description: >-
       Organization that is expected to receive some amount of the product.
-    type: object
+    $ref: ./Organization.yml
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/common/IntentToImport.yml:38]: Same here, declare an object as a subset of organization.

**Review by BenjaminMoe — CHANGES_REQUESTED** [review]
Request: Narrow definition for organization. You can declare a subset in the `snippets` folder.

**Review by BenjaminMoe — APPROVED** [review]

File: docs/openapi/components/schemas/credentials/BillOfLadingCredential.yml [hunk 3]
```diff
@@ -354,9 +354,9 @@ example: |-
     },
     "proof": {
       "type": "Ed25519Signature2018",
-      "created": "2022-11-29T07:37:53Z",
+      "created": "2022-12-20T15:30:19Z",
```
Comment (nissimsan) [inline docs/openapi/components/schemas/credentials/BillOfLadingCredential.yml:357]: @ipbyrne, you are re-signing all credentials for no reason. You shouldn't be touching the credentials which aren't actually part of the PR's scope. It makes it hard to understand the actual impact of the PR and causes unnecessary merge conflicts.
Please revert the re-signing of credentials which aren't needed. 
Next time, just run the `regenerate` script, it takes care of all this.

**Review by nissimsan — CHANGES_REQUESTED** [review]

File: docs/openapi/components/schemas/credentials/BillOfLadingCredential.yml (hunk 3 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/credentials/BillOfLadingCredential.yml:357]: Beyond the unnecessary resigns, the PR is great

**Pull request closed without merging**